packages/client/src/components/notebook/VariablePreview.tsx

```diff
@@ -151,12 +161,14 @@ export const VariablePreview = observer((props: VariablePreviewProps) => {
                     </Typography>
                     <Typography variant="body2">{variable.type}</Typography>
                 </Stack>
-                <Stack direction="row">
-                    <Typography variant="body2" fontWeight="bold">
-                        Value:{' '}
-                    </Typography>
-                    <Typography variant="body2">{val}</Typography>
-                </Stack>
+                {val !== null && (
```

Comment:
what prompted this check?

Reply:
This check is specifically used to remove showing the preview variable value twice, before it would show the editor view of the variable and then then a typography of the variable when you hovered over. 

Now it will only show the json-viewer if the value is an object. Which is what was requested for all objects.